include/domain_bridge/process_cmd_line_arguments.hpp

```diff
@@ -52,6 +52,12 @@ print_help()
     "                             This overrides any mode set in the YAML file." << std::endl <<
     "                             Defaults to '" << kNormalModeStr << "'." <<
     std::endl <<
+    "    --wait-for-subscription true|false  Will wait for an available subscription before"
+    " bridging a topic. Default is false, unless overridden in yaml file or here." <<
```

Comment:
nitpick: for consistency with other argument helps strings
```suggestion
    " bridging a topic. This overrides any value set in the YAML file. Defaults to false." <<
```

Reply:
Fixed in https://github.com/ros2/domain_bridge/pull/52/commits/ad1dd930448c2b491a14cba6aaa70f2c6534b3a5